mail/muchsync/Portfile

```diff
@@ -17,9 +17,9 @@ long_description    Muchsync brings Notmuch to all of your computers by \
 homepage            http://www.muchsync.org/
 master_sites        http://www.muchsync.org/src/
```

Comment:
can `homepage` and `master_sites` use "https"?

Reply:
What advantage would HTTPS have here? There is nothing worth protecting in transit, and downloaded files are checksum-verified.

Reply:
what is the disadvantage, we’ve been changing to ”https” wherever possible

Reply:
The disadvantage is wasting computing cycles (read energy) on completely unnecessary encryption and decryption. Using HTTPS over HTTP to download publicly available source code is environmentally harmful, and I therefore choose not to do it.